main.js

```diff
@@ -26,7 +26,10 @@ class Plugin {
 
 		addEvents.call(this)
 
-		if (!this.currentSettings.disable) {
+		if (
+			!this.currentSettings.disable ||
+			this.currentSettings.disable === 'auto'
```

Comment:
Est-ce qu'il ne faudra pas tester le `itemsNumber` ici ?

Reply:
Tu ne testes pas le `typeof` ici comme fait ailleurs ?

```js
typeof this.currentSettings.disable === "string"
```

Reply:
> Est-ce qu'il ne faudra pas tester le itemsNumber ici ?

En effet, je peux rajouter une vérification que la variable est bien supérieure à 0 (voire supérieure à 1, puisque techniquement si on n'a qu'un seul élément ça ne sert à rien d'implémenter un carousel dessus. Qu'est-ce que tu en penses ?)
> Tu ne testes pas le typeof ici comme fait ailleurs ?

Effectivement, je crois qu'il faut même que je check si le type est "boolean" pour le `!this.currentSettings.disable`, parce que si par exemple la variable n'est pas saisie, on veut quand même lancer le carousel